docs/instrumentation-list.yaml

```diff
@@ -6859,12 +6859,45 @@ libraries:
         attributes: []
   jsp:
   - name: jsp-2.3
+    display_name: JSP (JavaServer Pages)
```

Comment:
Actually the instrumentation name is a bit misleading. It really applies only to jasper, which is the jsp engine used by tomcat and all (except liberty?) open source app servers. Weblogic and websphere have their own jsp engines. The view rendering part should work for all jsp engines, but the compilation part will probably make muzzle reject the instrumentation for non-jasper implementations. This is just a FYI.

Reply:
😅